package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "cuttle",
   "description": "The deepest card game under the sea",
-  "version": "4.5.0",
+  "version": "4.5.1",
```

Comment:
So I've been continuing to ponder semver and I think it might make sense to essentially say:
1. bug fixes, refactoring, dep upgrades, and doc changes are patch versions
2. user facing features are minor versions
3. sweeping architectural changes (i.e. the vue3 upgrade, reworking the essential database architeture, etc) are major versions 
So this would be a patch, as you've got it. What do you think of that convention?

Reply:
I think it's totally fine, and ok to break a little out of the norm. I agree with that assessment for the most part, though. I will say if you're just changing a css var or visual bug, I can't imagine that's anything other than a patch version.